README.md

```diff
@@ -36,14 +36,16 @@ Releases can be found on the [release list][releases].
 
 ## System requirements
 
-- A not too ancient Linux, Windows 10 or later, or macOS 10.15 or later.
+- A not too ancient Linux (usually, only the lastest releases of popular distributions
```

Comment:
latest*

I think this advice works well in practice but it make it sound like 1) we only support certain distros and 2) there is some fixed point when it becomes unsupported.
You could probably use mpv fine on Ubuntu 18.04 if you used mpv-build.
at least I'd remove the "popular"